src/components/slider/slider.scss

```diff
@@ -49,16 +74,22 @@ md-slider {
 
   height: $slider-height;
   position: relative;
-  display: block;
   margin-left: 4px;
   margin-right: 4px;
   padding: 0;
+  justify-content: center;
+  display: flex;
```

Comment:
Could this be moved to a wrapper? I'm concerned users might attempt to make this block/inline and it would mess up your justification.